docs/production-deployment/cloud/metrics/index.mdx

```diff
@@ -27,8 +27,18 @@ tags:
   - temporal-cloud
 ---
 
-Beyond the [metrics](/references/sdk-metrics) provided by the Temporal SDKs, some key metrics exist only in Temporal Cloud.
-You can use your own observability tool to query an endpoint and review Namespace metrics.
+Temporal offers two distinct sources of metrics: [Cloud/Server Metrics](/production-deployment/cloud/metrics/reference) and [SDK Metrics](/references/sdk-metrics).
+Each source provides options for levels of granularity and filtering, monitoring-tool integrations, and configuration.
+Before implementing observability for Temporal Cloud, decide what you need to measure for your use case. There are two primary use cases for metrics:
+- To measure the health and performance of Temporal-backed applications and key business processes.
+- To measure the health and performance of Temporal infrastructure and user provided infrastructure in the form of Temporal Workers and Temporal Clients.
+
+When measuring the performance of Temporal-backed applications and key business processes, you should rely on Temporal SDK metrics as a source of truth.
```

Comment:
This says "you should rely on" and then the next sentence almost contradicts that. Missing word? Intentional?

Reply:
Hm, does it contradict? The point is that SDK metrics are useful for measuring how your own applications perform -- it reads well to me. How can I make it clearer?

Reply:
@jlacefie We chatted about this and docs team sounds like they'd like one more clarifying sentence here -- my reading is "If you want to know how your applications behave and how they experience latency -- the SDK metrics are the right ones to use.
 Cloud/Service metrics are more authoritative, but SDK metrics are more faithful to how your applications are actually behaving." Is that OK with you?